### PR TITLE
Add jupyterlab_widgets package

### PR DESCRIPTION
**Pull request by jasongrout**
CC @conda-forge/help-python

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jasongrout** [comment]
> Feedstock with the same name exists in conda-forge.

I can't find this feedstock: https://github.com/conda-forge/jupyterlab_widgets-feedstock gives a 404.

**Comment by jasongrout** [comment]
> * Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)

This package does contain bundled javascript, similar to JupyterLab. I think Webpack does package license files for the bundled Javascript in the `static/` directory of this package.

**Comment by jasongrout** [comment]
To be clear, here is the listing of files in the Javascript bundle:

```
1.6c3ab69171002cbfa04f.js
18.82ca398c3062f7a98981.js
243.6c384ff2649ef572a18a.js
272.65ab6d5670f13e2b0d62.js
279.aa88a78c8bf62c65db54.js
466.65c8606b1df82cee6ca5.js
523.1c76509bd5efdbac96d3.js
638.f3e5e34a28f3334d4f08.js
638.f3e5e34a28f3334d4f08.js.LICENSE.txt
803.b7b75bd6e7977a648c67.js
86.3e079cbb8e07ebac20cc.js
95.b5a5ff10000a6051fa99.js
95.b5a5ff10000a6051fa99.js.LICENSE.txt
imports.css
package.json
remoteEntry.f15bc08db7d10d18f84e.js
```
The two license files are:
```
% cat 638.f3e5e34a28f3334d4f08.js.LICENSE.txt
/*!
 * Sizzle CSS Selector Engine v2.3.4
 * https://sizzlejs.com/
 *
 * Copyright JS Foundation and other contributors
 * Released under the MIT license
 * https://js.foundation/
 *
 * Date: 2019-04-08
 */

/*!
 * jQuery JavaScript Library v3.4.1
 * https://jquery.com/
 *
 * Includes Sizzle.js
 * https://sizzlejs.com/
 *
 * Copyright JS Foundation and other contributors
 * Released under the MIT license
 * https://jquery.org/license
 *
 * Date: 2019-05-01T21:04Z
 */
% cat 95.b5a5ff10000a6051fa99.js.LICENSE.txt
/*!
 * jQuery UI Keycode 1.12.1
 * http://jqueryui.com
 *
 * Copyright jQuery Foundation and other contributors
 * Released under the MIT license.
 * http://jquery.org/license
 */

/*!
 * jQuery UI Mouse 1.12.1
 * http://jqueryui.com
 *
 * Copyright jQuery Foundation and other contributors
 * Released under the MIT license.
 * http://jquery.org/license
 */

/*!
 * jQuery UI Slider 1.12.1
 * http://jqueryui.com
 *
 * Copyright jQuery Foundation and other contributors
 * Released under the MIT license.
 * http://jquery.org/license
 */

/*!
 * jQuery UI Widget 1.12.1
 * http://jqueryui.com
 *
 * Copyright jQuery Foundation and other contributors
 * Released under the MIT license.
 * http://jquery.org/license
 */
```

**Comment by mariusvniekerk** [comment]
Could this package be noarch? or is the location of the js bits platform dependent?

File: recipes/jupyterlab_widgets/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+test:
+  imports:
+    - jupyterlab_widgets
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ipython/ipywidgets
```
Comment (jtpio) [inline recipes/jupyterlab_widgets/meta.yaml:35]: We might as well use the URL the `jupyter-widgets` organization?

```suggestion
  home: https://github.com/jupyter-widgets/ipywidgets
```

**Comment by jasongrout** [comment]
> Could this package be noarch?

Oh, right, yes, absolutely! Thanks for bringing this up. It should be noarch.

**Comment by jasongrout** [comment]
uh, not sure if that comment works here :).

File: recipes/jupyterlab_widgets/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: jupyterlab_widgets
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jupyterlab_widgets/jupyterlab_widgets-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  noarch: python
```
Comment (jasongrout) [inline recipes/jupyterlab_widgets/meta.yaml:13]: > Could this package be noarch?

@mariusvniekerk - doesn't this mean the package is noarch? Or is there more that we need to put in somewhere?

**Comment by jasongrout** [comment]
> This package does contain bundled javascript, similar to JupyterLab. I think Webpack does package license files for the bundled Javascript in the `static/` directory of this package.

I hand-copied the relevant license files into the main license file in https://github.com/jupyter-widgets/ipywidgets/pull/2993, releasing in alpha 5.

**Comment by jasongrout** [comment]
With the bump to alpha 5, the license distribution issues are resolved.

I think this is ready for review/merge now.

Thanks!

File: recipes/jupyterlab_widgets/meta.yaml (hunk 2 above)
Comment (mariusvniekerk) [reply recipes/jupyterlab_widgets/meta.yaml:13]: Nope this is sufficient

**Review by mariusvniekerk — APPROVED** [review]

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**